### PR TITLE
Css ecommerce page

### PR DESCRIPTION
**Pull request by jdfenlason**
Screen Shot 2021-03-18 at 7 46 29 PM

### CODE REVIEW

File: css-ecommerce-page/styles.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  display: flex;
+  flex-wrap: wrap;
+}
+
+@font-face {
+  font-family: 'DM Sans';
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+}
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css:19]: This can be removed, the link element for Google Fonts is already doing this for you.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far! Please see my note. Additionally, only the header of your document is using the right font family, make sure you apply that font family to all text on the page. Please make sure to attach an updated screenshot once you push up your changes!

**Comment by jdfenlason** [comment]
Screen Shot 2021-03-19 at 3 23 10 PM
I quadrupled checked to make sure it was inherited this time.  Definitely needed that exercise earlier!

**Review by codingcodymiller — APPROVED** [review]
Great work, man!!

**Pull request merged**